graphdatascience/system/system_endpoints.py

```diff
@@ -39,6 +81,10 @@ def is_licensed(self) -> bool:
     def debug(self) -> DebugProcRunner:
         return DebugProcRunner(self._query_runner, f"{self._namespace}.debug", self._server_version)
 
+    @property
+    def config(self) -> DebugProcRunner:
+        return DebugProcRunner(self._query_runner, f"{self._namespace}.config", self._server_version)
```

Comment:
```suggestion
        return DefaultsAndLimitsEndPoints(self._query_runner, f"{self._namespace}.config", self._server_version)
```
right?

Reply:
After config, you can only use `list` or `set` so the `DebugProcRunner` should be correct. checkstyle also complained there

Reply:
Actually I think this should be removed. `config` is not a direct endpoint as it's prefixed by `alpha`. You actually never use it

Reply:
My thinking was, that it gives a better auto-completion.

But then as we also dont have endpoints for individual algorithms, this probably falls into a similar category.
Will remove it :+1:

Reply:
It doesn't give any autocompletion as you never use it. You would need to write `gds.config.defaults....` since it's a _direct endpoint_, but that's not gonna work since the `DebugProcRunner` doesn't have a `defaults` attribute. Moreover, since `config` is not preceded by `alpha` the endpoint is not correct. Actually it does suggest `gds.config...` for autocompletion but that's misleading

Reply:
thanks for the clarification!